ui/admin/tests/e2e/helpers/boundary-cli/accounts.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+const { execSync } = require('child_process');
+
+/**
+ * Creates a new password account
+ * @param {string} authMethodId ID of the auth-method that the new account will be created for.
+ * @returns {Promise<string>} new account's ID
+ */
+export async function createPasswordAccountCli(authMethodId) {
```

Comment:
I want to mention these are actually ESM exports and not commonJS, but we're still importing in commonJS style throughout (i.e. using `require`). I just realized, was there any reason we needed to do that? Can all these e2e tests be swapped over to just use ESM imports/exports?

I'm not sure if the node modules will have any issue (e.g. `import { execSync } from 'node:child_process'`) but if they do, maybe we can even just swap over whole sale to ESM in admin UI now and declare in the `package.json` the type as `module`.

Reply:
@ZedLi I was just following an example set out here: https://github.com/hashicorp/boundary-ui/pull/2407/files. Happy to change to whatever is appropriate.

Reply:
Can we try switching over to ESM and see if we can get any issues? That would be much nicer and I'm not sure if there was any particular reason we needed to use commonJS imports

Reply:
Yup. Will try it out.

Reply:
@ZedLi Attempted to make some updates per your suggestion. Let me know if it's aligned with what you're thinking. Still need to test and make sure I didn't break anything.

Reply:
Ah right. I forgot about this, but last time I tried changing some of these imports, it would break the ember tests. It looks like it's doing it again. Do you have any insight to that?

Reply:
Yeah the main issue is we don't declare what type of module system to use so it will default to commonJS. The issue then is that the tests now use ESM which don't know how to properly import/export so using `.mjs` forces it. 

I was hoping we could force it to use ESM by changing the type in package.json but it seems like some packages have some interoperability issues. We would have to force them to `.cjs` through the whole chain which is a bigger headache than it's worth so I think just using `.mjs` in the e2e tests is easier.